cfgov/v1/jinja2/v1/_includes/molecules/form-field-with-button.html

```diff
@@ -0,0 +1,49 @@
+{# ==========================================================================
+
+   form-field-with-button.render()
+
+   ==========================================================================
+
+   Description:
+
+   Creates a form field with text when given:
+
+   settings:  An object of used to customize the markup.
+
+   settings.field.label:       The label for the input.
+
+   settings.field.name:        The name/id for the input.
+
+   settings.field.name:        The type of input.
+
+   settings.field.placeholder: The placeholder used for the input.
+
+   settings.btn_text:          The text to use for the submit button.
+
+   ========================================================================== #}
+
+{% macro render(settings) %}
+<div class="m-form-field-with-button">
+    <div class="form-group">
+        <label for="{{ settings.field.name }}">
+            {{ settings.field.label }}
+        </label>
+        <input id="field.name"
+               type="{{ settings.field.type }}"
+               name="{{ settings.field.name }}"
+               placeholder="{{ settings.field.placeholder }}"
+               required
+               class="m-form-field-with-button_field">
+    </div>
+    <p class="short-desc">
+        The information you provide will permit the Consumer Financial
+        Protection Bureau to process your request or inquiry.
+        <a href="/sub-pages/privacy-policy/">See More</a>
+    </p>
```

Comment:
This notice is specific to email subscribe forms and probably shouldn't be part of a general form-field-with-button molecule. Additionally, the URL will need to be a passable setting, because for the moment, different offices have different Privacy Act statements.